src/py/flwr/server/app.py

```diff
@@ -226,21 +234,27 @@ def run_driver_api() -> None:
     # Initialize StateFactory
     state_factory = StateFactory(args.database)
 
-    # Start server
-    grpc_server: grpc.Server = _run_driver_api_grpc(
-        address=args.driver_api_address,
-        state_factory=state_factory,
-    )
+    parsed_address = parse_address(args.driver_api_address)
+    if parsed_address:
```

Comment:
Can we reduce the "blast radius" of this change by doing the check + `sys.exit` here (and then leave the remainder of this function unchanged / unindented)?